app/liquid_tags/user_subscription_tag.rb

```diff
@@ -251,12 +282,15 @@ class UserSubscriptionTag < LiquidTagBase
     // Handle API responses
     // ***************************************
     function handleSubscription() {
+      hideConfirmationModal(); // Close the modal once the user has confirmed.
```

Comment:
This is a change from what we previously did, but I think it makes more sense to do this. With higher latency requests, this modal just stays open, and it's confusing because I think a user would expect that clicking "confirm" would close the modal (rather than them having to click "confirm" and then _also_ click the "X" of the modal to close it).